src/core/qgsdiagramrenderer.h

```diff
@@ -606,11 +682,30 @@ class CORE_EXPORT QgsDiagramRenderer
 
     //! Marker symbol to use in size legends
     std::unique_ptr< QgsMarkerSymbol > mSizeLegendSymbol;
+
+    /**
+     * The label used as title or the size attribute
+     * @brief mSizeLabel
+     */
+    QString mSizeLabel;
```

Comment:
Member variables are not part of the public API. It's ok to document them but not required.